test/Microsoft.Azure.SignalR.Protocols.Tests/ServiceProtocolFacts.cs

```diff
@@ -497,15 +547,6 @@ public static IEnumerable<object[]> TestParseOldData
                 name: "UserLeaveGroupWithTracingId",
                 message: new UserLeaveGroupMessage("conn11", "group2", tracingId: 1234L),
                 binary: "lBGmY29ubjExpmdyb3VwMoEBzQTS"),
-            new ProtocolTestData(
-                name: "GroupBroadcastWithTracingId",
-                message: new GroupBroadcastDataMessage("group3",
-                    new Dictionary<string, ReadOnlyMemory<byte>>
-                    {
-                        ["json"] = new byte[] {6, 7, 1, 2, 3, 4, 5},
-                        ["messagepack"] = new byte[] {7, 1, 2, 3, 4, 5, 6}
-                    }, tracingId: 1234L),
-                binary: "lQ2mZ3JvdXAzkIKkanNvbsQHBgcBAgMEBattZXNzYWdlcGFja8QHBwECAwQFBoEBzQTS"),
```

Comment:
why remove?

Reply:
reorder to merge with other `GroupBroadcastDataMessage` messages